console/nginx_conf.go

```diff
@@ -22,7 +22,7 @@ var NginxConf = `
 
 worker_processes auto;
 error_log /var/log/nginx/error.log;
-pid /run/nginx.pid;
+pid /var/lib/nginx/tmp/nginx.pid;
```

Comment:
```suggestion
pid /var/lib/nginx/run/nginx.pid;
```
can't we keep naming same as original ?